Library/Formula/td.rb

```diff
@@ -0,0 +1,49 @@
+require "language/go"
+
+class Td < Formula
+  homepage "https://github.com/Swatto/td"
+  revision 1
+
+  stable do
```

Comment:
Probably doesn't need to be nested inside a `stable do` block unless stable has patches, or special deps/resources.